### PR TITLE
gulpfile linting + looser .jscsrc

### PR DESCRIPTION
**Pull request by megawac**
Adds `crockford` style requirements to `curlys` (it's annoying to require em for case statements)

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -8,5 +8,15 @@
...
+
+  "requireCurlyBraces": [
+      "if",
+      "else",
+      "for",
+      "while",
+      "do",
+      "try",
+      "catch"
+  ]
```
Comment (jamesplease) [inline .jscsrc]: Remove (or edit) L5?

Comment (megawac) [inline .jscsrc]: Good catch

**Comment by jamesplease** [comment]
lgtm. Shall I merge?

**Comment by megawac** [comment]
Good to go IMO

File: .jscsrc [hunk 2]
```diff
@@ -2,11 +2,20 @@
...
-  "requireCurlyBraces": true,
   "requireCommaBeforeLineBreak": true,
   "requireOperatorBeforeLineBreak": true,
   "disallowTrailingWhitespace": true,
   "disallowMultipleLineStrings": true,
   "disallowMixedSpacesAndTabs": true,
-  "disallowEmptyBlocks": true
-}
+  "disallowEmptyBlocks": true,
+
```
Comment (jamesplease) [inline .jscsrc]: Delete this line?

Comment (megawac) [inline .jscsrc]: Why don't we just take one of the reasonable presets

https://github.com/jscs-dev/node-jscs/tree/master/presets

Comment (jamesplease) [inline .jscsrc]: Sure. I'm not sure which is closest to my own coding style, though. I know I dislike airbnb's, and I worked briefly w. Google's and that seemed fine. Any thoughtz?

Comment (megawac) [inline .jscsrc]: IDK, this is a case where yeoman generators are sweet - let me glance over the files

**Comment by megawac** [comment]
Changed it to a slightly amended version of good old `Crockford` style
- amended requiring `function()` instead of `function ()`
- esnext
- space: 2

**Comment by jamesplease** [comment]
Hmmm. Can you use [the preset option](https://github.com/marionettejs/backbone.marionette/blob/master/.jscsrc#L2) to set the defaults, instead of copy+pasting?

**Comment by megawac** [comment]
Didn't know that was a thing

File: .jscsrc [hunk 3]
```diff
@@ -1,12 +1,7 @@
...
-  "requireCurlyBraces": true,
-  "requireCommaBeforeLineBreak": true,
-  "requireOperatorBeforeLineBreak": true,
-  "disallowTrailingWhitespace": true,
-  "disallowMultipleLineStrings": true,
-  "disallowMixedSpacesAndTabs": true,
-  "disallowEmptyBlocks": true
-}
+  "preset": "google",
+
```
Comment (jamesplease) [inline .jscsrc]: remove space plz

Comment (megawac) [inline .jscsrc]: k

**Pull request merged**